vip-helpers/vip-newrelic.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Disable New Relic's browser metrics
+ *
+ * Removes NR's JavaScript for tracking browser metrics, including page load times, Apdex score, and more.
+ *
+ * Must be called at or before the `template_redirect` action.
+ */
+function wpcom_vip_disable_new_relic_js() {
+	if ( did_action( 'template_redirect' ) && ! doing_action( 'template_redirect' ) ) {
+		_doing_it_wrong( __FUNCTION__, 'New Relic&#8217;s browser tracking can only be disabled at or before the `template_redirect` action.', '1.0' );
+		return;
+	}
+
+	if ( function_exists( 'newrelic_disable_autorum' ) ) {
+		newrelic_disable_autorum();
+	}
+}
+
+/**
+ * Add the exact URI to NewRelic tracking but only if we're not in the admin
+ */
+function wpcom_vip_add_URI_to_newrelic(){
+	if ( ! is_admin() && function_exists( 'newrelic_add_custom_parameter' ) ){
+		newrelic_add_custom_parameter( 'REQUEST_URI', isset( $_SERVER['REQUEST_URI'] ) ? $_SERVER['REQUEST_URI'] : '' );
```

Comment:
Should we use `newrelic_capture_params` ([docs](https://docs.newrelic.com/docs/agents/php-agent/php-agent-api/newrelic_capture_params)) instead?